app/views/gitter/chat_button.erb

```diff
@@ -0,0 +1,17 @@
+<% if control.language.downcase == 'ruby' %>
+  <div class="gitter-chat-button-frame">
+    <div class="gitter-chat-button">
+      Open Chat: Enjoy the <%= control.language.capitalize %> conversation!
+    </div>
+  </div>
+
+  <script>
+    ((window.gitter = {}).chat = {}).options = {
+      room: 'exercism/xruby',
```

Comment:
Right, the only place that would actually know the real value is the `config.json` file, but right now we get the config data via x-api. We would be able to switch on whether the `gitter` config value is present or not in order to show the information.